[2024-11-05T17:13:03.693+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T17:13:03.858+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.specifications_scraping_tasks.gpu_scrape_task manual__2024-11-05T17:13:02.967561+00:00 [queued]>
[2024-11-05T17:13:03.869+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.specifications_scraping_tasks.gpu_scrape_task manual__2024-11-05T17:13:02.967561+00:00 [queued]>
[2024-11-05T17:13:03.869+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-05T17:13:03.886+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): specifications_scraping_tasks.gpu_scrape_task> on 2024-11-05 17:13:02.967561+00:00
[2024-11-05T17:13:03.890+0000] {standard_task_runner.py:72} INFO - Started process 1664 to run task
[2024-11-05T17:13:03.894+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'specifications_scraping_tasks.gpu_scrape_task', 'manual__2024-11-05T17:13:02.967561+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp3wra_4gn']
[2024-11-05T17:13:03.896+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask specifications_scraping_tasks.gpu_scrape_task
[2024-11-05T17:13:03.965+0000] {task_command.py:467} INFO - Running <TaskInstance: test_dag.specifications_scraping_tasks.gpu_scrape_task manual__2024-11-05T17:13:02.967561+00:00 [running]> on host 45fe9341983d
[2024-11-05T17:13:04.054+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='veil' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='specifications_scraping_tasks.gpu_scrape_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T17:13:02.967561+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-05T17:13:02.967561+00:00'
[2024-11-05T17:13:04.054+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T17:13:04.066+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T17:13:04.067+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /app/scraper && scrapy crawl gpu_spider -O ../temp/gpu_data.json']
[2024-11-05T17:13:04.073+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T17:27:31.478+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-11-05T17:27:31.479+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-05T17:27:31.481+0000] {process_utils.py:132} INFO - Sending 15 to group 1664. PIDs of all processes in the group: [1682, 1683, 1700, 1844, 1875, 1938, 2052, 2064, 2084, 2092, 1664]
[2024-11-05T17:27:31.482+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1664
[2024-11-05T17:27:31.482+0000] {taskinstance.py:3092} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-11-05T17:27:31.483+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2024-11-05T17:27:31.512+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T17:27:31.514+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2064, status='terminated', started='17:13:06') (2064) terminated with exit code None
[2024-11-05T17:27:31.611+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1700, status='terminated', started='17:13:04') (1700) terminated with exit code None
[2024-11-05T17:27:31.611+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2052, status='terminated', started='17:13:06') (2052) terminated with exit code None
[2024-11-05T17:27:31.612+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2084, status='terminated', started='17:13:06') (2084) terminated with exit code None
[2024-11-05T17:27:31.613+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1664, status='terminated', exitcode=0, started='17:13:03') (1664) terminated with exit code 0
[2024-11-05T17:27:31.614+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=2092, status='terminated', started='17:13:06') (2092) terminated with exit code None
[2024-11-05T17:27:31.615+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1844, status='terminated', started='17:13:04') (1844) terminated with exit code None
[2024-11-05T17:27:31.712+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1875, status='terminated', started='17:13:05') (1875) terminated with exit code None
[2024-11-05T17:27:31.713+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1938, status='terminated', started='17:13:05') (1938) terminated with exit code None
[2024-11-05T17:27:34.012+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-11-05T17:27:41.566+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1683, status='terminated', started='17:13:04') (1683) terminated with exit code None
[2024-11-05T17:27:41.623+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1682, status='terminated', started='17:13:03') (1682) terminated with exit code None
